package.json

```
@@ -39,7 +39,7 @@
   "peerDependencies": {
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "styled-components": "^4.3.2"
+    "styled-components": "^5.2.1"
```

We probably don't need to remove support for `4`:
```suggestion
    "styled-components": "^4.3.2 || ^5.0.0"
```